app/src/main/java/io/homeassistant/companion/android/HomeAssistantBootReceiver.kt

```diff
@@ -0,0 +1,18 @@
+package io.homeassistant.companion.android
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import io.homeassistant.companion.android.sensors.SensorWorker
+
+/**
+ * This class will receive a broadcast intent when the device boots up. This allows us to start the sensor worker
+ * even if the user never starts the HomeAssistant app.
+ */
+class HomeAssistantBootReceiver : BroadcastReceiver() {
+    override fun onReceive(context: Context, intent: Intent) {
+        if (intent.action == Intent.ACTION_BOOT_COMPLETED) {
+            SensorWorker.start(context)
+        }
+    }
+}
```

Comment:
Now that this is the only boot reciever we need to send a broadcast to restart location tracking.
https://github.com/home-assistant/home-assistant-android/blob/master/app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt#L141